packages/wonder-blocks-dropdown/components/single-select.js

```diff
@@ -271,24 +345,41 @@ export default class SingleSelect extends React.Component<Props, State> {
     }
 
     render() {
-        const {alignment, dropdownStyle, light, style} = this.props;
-        const items = this.getMenuItems();
-        const opener = this.renderOpener(items.length);
+        const {
+            alignment,
+            children,
+            dropdownStyle,
+            isFilterable,
+            light,
+            style,
+        } = this.props;
+        const {searchText} = this.state;
+        const allChildren = React.Children.toArray(children).filter(Boolean);
+        const filteredItems = this.getMenuItems(allChildren);
+        const opener = this.renderOpener(allChildren.length);
 
         return (
             <DropdownCore
                 role="listbox"
                 alignment={alignment}
-                dropdownStyle={[selectDropdownStyle, dropdownStyle]}
+                dropdownStyle={[
+                    isFilterable && filterableDropdownStyle,
+                    selectDropdownStyle,
+                    dropdownStyle,
+                ]}
                 initialFocusedIndex={this.selectedIndex}
-                items={items}
+                items={[...this.getSearchField(), ...filteredItems]}
                 keyboard={this.state.keyboard}
                 light={light}
                 onOpenChanged={this.handleOpenChanged}
                 open={this.state.open}
                 opener={opener}
                 openerElement={this.openerElement}
                 style={style}
+                onSearchTextChanged={
+                    isFilterable ? this.handleSearchTextChanged : null
+                }
+                searchText={isFilterable ? searchText : ""}
```

Comment:
This was already part of `dropdown-core.js` because J added this when she created the same functionality for the `MultiSelect` functionality. 
You can see the PR here: https://github.com/Khan/wonder-blocks/pull/479